book_module.rb

```diff
@@ -0,0 +1,19 @@
+require './book'
+
+module NewBook
+  def create_book
+    print 'Title:'
+    title = gets.chomp
+    print 'Author:'
+    author = gets.chomp
+    book = Book.new(title: title, author: author)
+    @books << book
+    puts 'Book created successfully'
+  end
```

Comment:
- [ ] good job on cleaning up the `main.rb` file 👍 however this method is not according to the "single responsibility principle" all requests for input and input validation should've been placed in classes/methods whose responsibilities are to request or validate input. I've put a small example below

```
def get_book_info
    print 'Title:'
    title = gets.chomp
    print 'Author:'
    author = gets.chomp
   
    puts 'Book created successfully'
    "and at the end return title and author"
end
def create_book
   book_info = get_book_info
    book = Book.new(title: title, author: author)
    @books << book
end
```

Reply:
Thank you for your valuable comment, we solved it as you can see in the image below
![bookmodule](https://user-images.githubusercontent.com/60292761/141951911-403c7d80-f99c-4862-9ea7-aa9f981103e8.PNG)
.